clang/test/Driver/modules-print-library-module-manifest-path.cpp

```diff
@@ -0,0 +1,38 @@
+// Test that -print-library-module-manifest-path finds the correct file.
+
+// REQUIRES: x86-registered-target
```

Comment:
```suggestion
// FIXME: ....
// REQUIRES: x86-registered-target
```

I feel better with a FIXME to remind us to remove this later.